apps/dg/libraries/slickgrid/plugins/slick.rowselectionmodel.js

```diff
@@ -34,10 +34,12 @@
 
     function wrapHandler(handler) {
       return function () {
+        var result;
```

Comment:
I added another commit which fixes the wrapHandler() in the DG.CaseTableRowSelectionModel. In my testing, removing Slick.RowSelectionModel seems not to have any effect, so I'm inclined to commit its removal as well.